packages/react/.storybook/preview.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { SlashIDProvider } from "../src/main";
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
+  },
+};
+
+export const decorators = [
+  (Story) => (
+    <SlashIDProvider
+      // @ts-ignore
+      oid={import.meta.env.VITE_ORG_ID}
+      themeProps={{ theme: "dark" }}
```

Comment:
I think here we should use the default (`auto`) but expose this setting via the [controls plugin](https://storybook.js.org/docs/react/essentials/controls) somehow.

It would be useful for people to know and see it can be changed easily, but should default to their OS/browser setting.

If controls plugin is too complicated, I would still suggest that we use `auto` at the very least.

Reply:
I tried to expose the theme setting but it seems that it's only possible for story component prop, so I'm not sure how to do it for `<SlashIDProvider>` since it's used as a wrapper for every story, any idea how to make it work?